### PR TITLE
Enhance E2E Tests - Http to Http Transfer

### LINKED ISSUE #207: Feat: Enhance E2E Tests - Http to Http Transfer
# Description
Enahance E2E tests to validate a successful http to http transfer via backend-service.
- Create a dummy content in backend service and get an url.
- Use above url as a http source while creating asset.
- Create requests in postman to initiate negotiation.
- Validate Negotiation state as `COMPLETED`.
- Initiate transfer with http destination as backend service.
- Validate Transfer State as `COMPLETED`.
- Validate http transfer via backend service.

Related to eclipse-tractusx/sig-release#316

### PR DESCRIPTION
**Pull request by ravinderkumarsap**
## Description:   Feat: Enhance E2E Tests - Http to Http Transfer #207
 
Please describe your PR: 
- What does this PR introduce: Enhanced E2E tests to validate for a successful http to http transfer via backend-service.
- Does it fix a bug? : NO
- Does it add a new feature? : No
- Is it enhancing documentation? : NO 

It follows the below step for end-to-end http testing

Created a dummy content in backend service and get an ID of Created Content.
Used above Created Content and Backend Service URL pass as a http source while creating asset.
Created requests in postman to initiate negotiation.
Validated Negotiation state as COMPLETED.
Initiated transfer with http destination as backend service.
Validated Transfer State as COMPLETED.
Validated http transfer via backend service.

Closes  #207

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by hemantxpatel** [comment]
Depends on PR #172.

File: mxd/postman/management_api_tests.json
Comment (hemantxpatel) [inline mxd/postman/management_api_tests.json:407]: Add `odrl:target` field inside permission object. We downgraded MXD version from latest to 0.6.0, so it will not work.

File: mxd/postman/management_api_tests.json
Comment (hemantxpatel) [inline mxd/postman/management_api_tests.json:456]: Field name is `state` instead of `edc:state`

File: mxd/postman/management_api_tests.json
Comment (hemantxpatel) [inline mxd/postman/management_api_tests.json:588]: Field name is `state` instead of `edc:state`

**Review by hemantxpatel — CHANGES_REQUESTED** [review]

File: mxd/postman/management_api_tests.json
Comment (ravinderkumarsap) [reply mxd/postman/management_api_tests.json:209]: I have removed the body part from Get All Policies.

File: mxd/postman/management_api_tests.json
Comment (ravinderkumarsap) [reply mxd/postman/management_api_tests.json]: I have updated from edc:state to state in all the required places.

File: mxd/postman/management_api_tests.json
Comment (ravinderkumarsap) [reply mxd/postman/management_api_tests.json]: I have updated from **edc:state** to **state** in all the required places.

**Review by ndr-brt — COMMENTED** [review]
I'm not really knowledgeable in postman format, I will merge this once @hemantxpatel will approve it

**Review by hemantxpatel — APPROVED** [review]
Verified running postman collection, looks good.

**Pull request merged**
